src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/TypeOperations.cs

```diff
@@ -156,13 +156,13 @@ public GetValueOrDefault(MethodInfo mi)
             }
 
             [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2077:UnrecognizedReflectionPattern",
-                Justification = "_defaultValueType is a ValueType. You can always create an instance of a ValueType.")]
+                Justification = "_defaultValueType is a ValueType. You can always get an uninitialized ValueType.")]
             public override int Run(InterpretedFrame frame)
             {
                 if (frame.Peek() == null)
                 {
                     frame.Pop();
-                    frame.Push(Activator.CreateInstance(_defaultValueType));
+                    frame.Push(RuntimeHelpers.GetUninitializedObject(_defaultValueType));
```

Comment:
There is no assertion in the ctor here that `_defaultValueType` is not nullable as there is in the ctor for `DefaultValueInstruction`. Would expect an assert in both places.